README.md

```diff
@@ -1,838 +1,42 @@
-    - STATUS: Unknown.
-- [putc()](https://linux.die.net/man/3/fputs).
-    - STATUS: Unknown.
-- [puts()](https://linux.die.net/man/3/fputs).
-    - STATUS: Unknown.
-- [remove()](https://linux.die.net/man/3/remove)
-    - STATUS: Basic testing implemented. Working.
-- [rewind()](https://linux.die.net/man/3/rewind).
-    - STATUS: Basic testing implemented. Working. 
-- [stat()](https://linux.die.net/man/2/stat)
-    - STATUS: Implemented. Working.
-    - STATUS: Not supported by ARMCC/IAR libc.
-- [tmpfile()](https://linux.die.net/man/3/tmpfile).
-    - STATUS: Not implemented.
-- [tmpnam()](https://linux.die.net/man/3/tmpnam). 
-    - STATUS: Not implemented.
-
-Supported directory related operations are as follows:
-
-- [closedir()](https://linux.die.net/man/3/closedir).
-    - STATUS: Implemented. Working.
-- [mkdir()](https://linux.die.net/man/3/mkdir). 
-    - STATUS: Basic testing implemented. Working.
-- [opendir()](https://linux.die.net/man/3/opendir). 
-    - STATUS: Implemented. Working.
-- [readdir()](https://linux.die.net/man/3/readdir).
-    - STATUS: Implemented. Working.
-- [remove()](https://linux.die.net/man/3/remove).
-    - STATUS: Basic testing implemented. Working.
-- [rename()](https://linux.die.net/man/3/rename).
-    - STATUS: Implemented. Not tested.
-- [rewinddir()](https://linux.die.net/man/3/rewinddir).
-    - STATUS: Implemented. Found not to work. Test case not present in repo.
-- [seekdir()](https://linux.die.net/man/3/seekdir).
-    - STATUS: Implemented. Found not to work. Test case not present in repo. 
-- [telldir()](https://linux.die.net/man/3/telldir).
-    - STATUS: Implemented. Found not to work. Test case not present in repo.
-
-### errno 
-
-Basic errno reporting is supported, tested and known to be working.
-
-
-# Related Projects Resources
-
-The following are related mbed storage projects and useful resources:
-
-- The [mbed-os](https://github.com/ARMmbed/mbed-os) main repository.
-- The [mbed-os-example-fat-filesystem](https://github.com/ARMmbed/mbed-os-example-fat-filesystem) repository.
-  This is an example project for the mbed OS FAT filesystem.
-- The [spiflash-driver](https://github.com/armmbed/spiflash-driver) repository.
-- The [i2ceeprom-driver](https://github.com/ARMmbed/i2ceeprom-driver.git) repository.
-- The [ci-test-shield](https://github.com/ARMmbed/ci-test-shield)  repository. This is the project describing 
-  the mbed-os Continuous Integration test shield, together with standard tests.
-- The [mbed-HDK](https://github.com/ARMmbed/mbed-HDK) repository containing Hardware Development Kit resources
-  including the schematics for the CI test shield. 
-- [POSIX File Interface ISO/IEC 9899:TC2 Documentation](http://www.eng.utah.edu/~cs5785/slides-f10/n1124.pdf).
-- [FATFS: Generic FAT File System Module used in mbed OS](http://elm-chan.org/fsw/ff/00index_e.html)
-
-
+	// Instantiate the SDBlockDevice by specifying the SPI pins connected to the SDCard
```

Comment:
Small nit: If you use the three ticks for code blocks, you can add the source type to get syntax highlighting. This helps quite a bit with readability:

    ``` cpp
    int main() {
        printf("blablabla");
    }
    ```


``` cpp
int main() {
    printf("blablabla");
}
```

https://help.github.com/articles/creating-and-highlighting-code-blocks/